### PR TITLE
Feature/new yaml file

### PR DESCRIPTION
**Pull request by bmoran311**

### CODE REVIEW

File: independent-work/bmoran/bmoran-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Build and run Dockerfile for bmoran
+
+on:
+    pull_request:
+        branches:
+          - master
+        path:
+          - 'individual-work/bmoran.**
```
Comment (eddie-knight) [inline independent-work/bmoran/bmoran-workflow.yaml:8]: missing apostraphe at end of this line

Comment (bmoran311) [reply independent-work/bmoran/bmoran-workflow.yaml]: Done, just re-pushed.

File: independent-work/bmoran/bmoran-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Build and run Dockerfile for bmoran
+
+on:
+    pull_request:
+        branches:
+          - master
+        path:
+          - 'individual-work/bmoran.**'
```
Comment (eddie-knight) [inline independent-work/bmoran/bmoran-workflow.yaml:8]: `'individual-work/bmoran/**'`

Comment (eddie-knight) [reply independent-work/bmoran/bmoran-workflow.yaml:8]: Also please correct this file location to be in `. github/workflows/bmoran-workflow.yaml`

**Comment by bmoran311** [comment]
This is obsolete

**Pull request closed without merging**